python/paddle/v2/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+list(REMOVE_ITEM TEST_OPS test_modified_huber_loss_op) # FIXME(qijun) https://github.com/PaddlePaddle/Paddle/issues/5184 
+list(REMOVE_ITEM TEST_OPS test_lstm_unit_op) # # FIXME(qijun) https://github.com/PaddlePaddle/Paddle/issues/5185
+list(REMOVE_ITEM TEST_OPS test_nce) # IXME(qijun) https://github.com/PaddlePaddle/Paddle/issues/7778
+list(REMOVE_ITEM TEST_OPS test_recurrent_op) # FIXME(qijun) https://github.com/PaddlePaddle/Paddle/issues/6152
+list(REMOVE_ITEM TEST_OPS test_cond_op) # FIXME(qijun): https://github.com/PaddlePaddle/Paddle/issues/5101#issuecomment-339814957
+list(REMOVE_ITEM TEST_OPS test_detection_output_op) # FIXME: detection_output_op will be rewritten. This unittest should be
+
+list(REMOVE_ITEM TEST_OPS op_test) # op_test is a helper python file, not a test
+list(REMOVE_ITEM TEST_OPS decorators) # decorators is a helper python file, not a test
+
+# test time consuming OPs in a separate process for expliot parallism
+list(REMOVE_ITEM TEST_OPS test_warpctc_op)
+py_test_modules(test_warpctc_op MODULES test_warpctc_op ENVS FLAGS_warpctc_dir=${WARPCTC_LIB_DIR})
+
+list(REMOVE_ITEM TEST_OPS test_dyn_rnn)
+py_test_modules(test_train_dyn_rnn MODULES test_dyn_rnn)
+
+list(REMOVE_ITEM TEST_OPS test_mul_op)
+py_test_modules(test_mul_op MODULES test_mul_op)
+
+list(REMOVE_ITEM TEST_OPS test_multihead_attention)
+py_test_modules(test_multihead_attention MODULES test_multihead_attention)
+list(REMOVE_ITEM TEST_OPS test_calc_gradient)
+py_test_modules(test_calc_gradient MODULES test_calc_gradient)
+list(REMOVE_ITEM TEST_OPS test_while_op)
+py_test_modules(test_while_op MODULES test_while_op)
+list(REMOVE_ITEM TEST_OPS test_lod_array_length_op)
+py_test_modules(test_lod_array_length_op MODULES test_lod_array_length_op)
+list(REMOVE_ITEM TEST_OPS test_reorder_lod_tensor)
+py_test_modules(test_reorder_lod_tensor MODULES test_reorder_lod_tensor)
+list(REMOVE_ITEM TEST_OPS test_profiler)
+py_test_modules(test_profiler MODULES test_profiler)
+list(REMOVE_ITEM TEST_OPS test_normalization_wrapper)
+py_test_modules(test_normalization_wrapper MODULES test_normalization_wrapper)
+list(REMOVE_ITEM TEST_OPS test_executor_and_mul)
+py_test_modules(test_executor_and_mul MODULES test_executor_and_mul)
+list(REMOVE_ITEM TEST_OPS test_assign_value_op)
+py_test_modules(test_assign_value_op MODULES test_assign_value_op)
+list(REMOVE_ITEM TEST_OPS test_array_read_write_op)
+py_test_modules(test_array_read_write_op MODULES test_array_read_write_op)
+list(REMOVE_ITEM TEST_OPS test_lod_rank_table)
+py_test_modules(test_lod_rank_table MODULES test_lod_rank_table)
+list(REMOVE_ITEM TEST_OPS test_weight_normalization)
+py_test_modules(test_weight_normalization MODULES test_weight_normalization)
+list(REMOVE_ITEM TEST_OPS test_conditional_block)
+py_test_modules(test_conditional_block MODULES test_conditional_block)
+
+# Allow to split the test so that developer can invoke a specific test.
+if(WITH_FAST_BUNDLE_TEST)
```

Comment:
It seems that we'd always want to run tests the fast way. How about we remove the CMake option `WITH_FAST_BUNDLE_TEST` by assuming it is true?

Reply:
When one test within the bundled test fails, user might want to only test that failed test instead of testing all tests in the bundle (takes a long time)?

Reply:
Good point!

Reply:
The PR passes now. I can think about your proposal as a followup